src/main/java/hudson/scm/SubversionSCM.java

```diff
@@ -2197,6 +2198,12 @@ public boolean isBrowserReusable(SubversionSCM x, SubversionSCM y) {
             return true;
         }
 
+        @Nonnull
+        @Override
+        public Permission getRequiredGlobalConfigPagePermission() {
+            return Jenkins.MANAGE;
```

Comment:
Changing this permission check is enough? there is no other permission check in the submitting, or in the jelly template?